webapp-django/crashstats/signature/static/signature/js/signature_tab_reports.js

```diff
@@ -104,14 +104,7 @@ SignatureReport.ReportsTab.prototype.getParamsForUrl = function () {
     }
 
     // Get the sort for the input.
-    var sortArr = this.$sortInputHidden.val().split(',');
-    if (sortArr.length === 1 && !sortArr[0]) {
-        sortArr = [];
-    }
-
-    if (sortArr) {
-        params._sort = sortArr;
-    }
+    params._sort = this.$sortInputHidden.val().trim().split(',');
```

Comment:
This is different from the code you had before. If the `$sortInputHidden` is empty, the `params._sort` will become `['']`.